Mod2/Labs/Week1/SQLCRUD.md

```diff
@@ -0,0 +1,50 @@
+## Lab Objectives
+* Practice using `SELECT` `UPDATE` `REMOVE` and `INSERT`
+
+** Instructor Note ** Students are going to build upon their work from yesterday's lab.  It would be helpful if they are in the same groups, if possible.  If students need to start from a fresh database setup, you can provide them with this script (it should work if their database is called 'Library').
+```SQL
+DROP TABLE books, patrons;
+-- this should be a list of their tables.
```

Comment:
Should we be more explicit on how `DROP TABLE` command works, and how they must change `books, patrons` to the actual names of the tables they created in the **Intro to Databases** lab?